script.js

```diff
@@ -1,48 +1,126 @@
 let startBtn = document.querySelector("#startButton");
 startBtn.addEventListener("click", playGame);
-//ash says we can use this as a global object
+
 let gameState = ["", "", "", "", "", "", "", "", ""];
+let currentPlayer = "x";
+const tiles = document.querySelectorAll(".tile");
```

Comment:
I don't mind globals being used to track game state, but I don't like that tiles is now used globally. Would be better to select the tiles in playGame and then pass those into your clickEvent function